releasenotes/notes/datadog-cast-pin-aff8991253932273.yaml

```
@@ -0,0 +1,12 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Restore the string slices read behavior while reading configuration in
+    the environment variables.
```

nit: a bit hard to understand for the non-initiated. Suggestion (not sure it's that better though):

```suggestion
    Restore support of JSON-formatted lists for configuration options passed as environment variables.
```